lib/bgpstream_elem.h

```diff
@@ -187,6 +200,12 @@ typedef struct bgpstream_elem {
    */
   bgpstream_elem_peerstate_t new_state;
 
+  /** Annotations
+   *
+   * Annotations from other libraries
+   */
+  bgpstream_annotations_t annotations;
+
```

Comment:
enclose the above with #ifdef ?

Reply:
no, let's go with your other suggestion above.